frontend/src/components/Spinner.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const Spinner = () => (
+  <svg
```

Comment:
It looks like [the request from the ticket](https://github.com/HHS/simpler-grants-gov/issues/1228#issuecomment-2045280590) was to use something more like: https://design.cms.gov/components/spinner/?theme=core . Was there another convo about that that didn't make it to the ticket?

A CSS solution would be lighter weight and would prob avoid the loading issue you noted. The CMS spinner is in CSS. There are some options here as well though they might have some cross-browser issues: https://css-loaders.com/spinner/ .

Reply:
Just put this in a CSS file: 
```
@use 'uswds-core' as *;
$spinner-size: units(4);

.grants-spinner {
  box-sizing: border-box;
  display: inline-block;
  position: relative;
  height: $spinner-size;
  width: $spinner-size;

  // Spinner icon pieces (full and partial circles)
  &::before,
  &::after {
    border-radius: 50%;
    border-style: solid;
    border-width: calc(100% / 8);
    box-sizing: border-box;
    content: "";
    display: block;
    height: 100%;
    left: 0;
    position: absolute;
    top: 0;
    width: 100%;
  }

  // Full circle
  &::before {
    border-color: currentColor;
    opacity: 0.2;
  }

  // Partial circle
  &::after {
    animation: spin 0.8s infinite linear;
    border-bottom-color: transparent;
    border-left-color: currentColor;
    border-right-color: transparent;
    border-top-color: transparent;
    transform: translateZ(0);
  }
}

@keyframes spin {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
```
…and use the `<span aria-label="Loading!" class="grants-spinner" role="progressbar" />` wherever we need it.

Reply:
Thanks I ended up using this!